addons/web/static/src/scss/form_view.scss

```diff
@@ -516,6 +516,11 @@
             }
         }
 
+        .o_field_widget_livechat_layout_colors {
```

Comment:
I would define this top-level... I know, I know, hierarchical selectors with SCSS looks good, but this is a pain to understand and maintain in reality, plus it forces us to handle CSS specificity, which is bad IMO